### PR TITLE
Router response alerts

### PR DESCRIPTION
**Pull request by rahulkumark**
**Fixes** # (*issue*)

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
## Checklist:
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added unit tests for the code
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: router/router.go [hunk 1]
```diff
@@ -218,6 +218,9 @@ func (rt *HandleT) workerProcess(worker *workerT) {
 			ErrorResponse: []byte(`{}`),
 		}
 
+		routerResponseStat := stats.NewRouterStat("router_response_counts", stats.CountType, rt.destName, respStatusCode)
+		routerResponseStat.Increment()
+
 		if isSuccessStatus(respStatusCode) {
 			atomic.AddUint64(&rt.successCount, 1)
 			eventsDeliveredStat.Increment()
```
Comment (SumanthPuram) [inline router/router.go:226]: Should we remove the delivered and failed stats?

**Review by SumanthPuram — COMMENTED** [review]
Should we remove earlier stats - eventsDelivered and failedStat ? CC: @sivashanmukh

File: router/router.go (hunk 1 above)
Comment (sivashanmukh) [reply router/router.go:226]: Yea. Old stats should be removed.

**Review by sivashanmukh — CHANGES_REQUESTED** [review]

**Review by sivashanmukh — APPROVED** [review]
LGTM

**Review by chandumlg — APPROVED** [review]

**Pull request merged**